src/librawrtc/ice_gatherer.c

```diff
@@ -1434,14 +1437,14 @@ static enum rawrtc_code add_candidate(
  * TODO: https://tools.ietf.org/html/draft-ietf-rtcweb-ip-handling-01
   */
 static bool interface_handler(
-        char const* interface, // not checked
+        char const* iface, // not checked
```

Comment:
I'd prefer `interface_` to avoid a name clash. Also, please add a comment somewhere in that code section that `interface` clashes and with what it clashes.